website_sale_tax_toggle/models/website.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2025 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Website(models.Model):
+    _inherit = "website"
+
+    default_tax_toggle = fields.Boolean(
```

Comment:
I already suggested you to change the name of the field, both for discoverability and for avoiding the prefix `default_`, that can be used for other purposes:

```suggestion
    tax_toggle_preactivated = fields.Boolean(
```